.github/workflows/ci-actions.yml

```diff
@@ -0,0 +1,249 @@
+name: CI
+
+on:
+  push:
+    paths-ignore:
+      - "README.md"
+  pull_request:
+    types: [assigned, opened, synchronize, reopened, ready_for_review, edited]
```

Comment:
Could we avoid having to list the events removing this `types` array?

Reply:
Removed `types`